addons/mail/static/src/models/message_action_list/message_action_list.js

```diff
@@ -83,15 +98,49 @@ function factory(dependencies) {
          * @param {CustomEvent} ev
          */
         onDeleteConfirmDialogClosed(ev) {
-            this.update({ showDeleteConfirm: false })
+            this.update({ showDeleteConfirm: false });
         }
 
+        /**
+         * Called when selection an emoji from the emoji popover (from the emoji
+         * button).
+         *
+         * @private
+         * @param {CustomEvent} ev
+         * @param {Object} ev.detail
+         * @param {string} ev.detail.unicode
+         */
+        onEmojiSelection(ev) {
+            this.message.addReaction(ev.detail.unicode);
```

Comment:
probably want to rename "unicode" here too.

Reply:
This is the existing event of the emoji popover, out of scope to rename it here.